src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+
+}
+
+.about__title {
+    margin-bottom: 5rem;
+    font-size: 3rem;
+    font-family: fantasy;
+}
+
+.about__description_container {
+    width: 100%;
```

Comment:
[Please Change]: I don't believe this has an effect; this and the corresponding div can likely be removed. Block elements should be full-width by default.